sound/soc/sof/sof-priv.h

```diff
@@ -464,6 +471,9 @@ struct snd_sof_dev {
 	wait_queue_head_t waitq;
 	int code_loading;
 
+	/* virtio for BE and FE */
+	bool is_vfe;
```

Comment:
what does 'for BE and FE' mean for a variable that's only 'fe'?

Reply:
@plbossart this is an original comment, I'll reword